spec/controllers/tas_controller_spec.rb

```diff
@@ -212,4 +212,47 @@
       end
     end
   end
+
+  context '#update' do
+    let(:params) do
+      {
+        user: {
+          grader_permission_attributes: {
+            manage_assessments: false,
+            manage_submissions: true,
+            run_tests: true
+          },
+          user_name: 'c6conley',
+          first_name: 'Mike',
+          last_name: 'Conley'
+        }
+      }
+    end
+    before do
+      post :create, params: params
```

Comment:
use a factory here please. These tests are meant to test the `update` method, not the `create` method.